client/src/components/LoginInput.js

```diff
@@ -0,0 +1,29 @@
+import styled from '@emotion/styled';
+
+const LoginInput = styled.input`
+  display: flex;
```

Comment:
It looks like you are setting more CSS than required. 
Is it really necessary to have a flexbox with everything centered and a text-align which is centered too for the placeholder? Maybe `text-align: center` is enough and you don't need the flexbox at all?
I recommend double-checking if you need all of these properties.

Reply:
Yes, `text-align: center` is enough. Thanks :)